migration/migrator/migrations/system/20210722131039_change_autograding_workers_json_permission.py

```diff
@@ -0,0 +1,26 @@
+"""Migration for the Submitty system."""
+
+from pathlib import Path
+import os
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    workers_json = str(Path(config.submitty['submitty_install_dir'], 'config', 'autograding_workers.json'))
+    os.chmod(workers_json, 0o660)
+    os.system("chown submitty_php {}".format(workers_json))
```

Comment:
This chown looks like it'll be overwritten by `INSTALL_SUBMITTY_HELPER.sh` doing `shutil.chown(full_file_name, 'root',DAEMON_GID)`?